HISTORY.md

```diff
@@ -9,6 +9,9 @@
 ### Performance Improvements
 * Try to avoid updating DBOptions if `SetDBOptions()` does not change any option value.
 
+### Public API change
```

Comment:
You can add  ### Behavior Changes and move the comment below it.

Reply:
Oh, sorry, I mixed up the two sections.